src/Tools/MicroBuild/run-pdbgit.ps1

```diff
@@ -40,6 +40,14 @@ try
     $count = 0
 
     foreach ($relativePath in $signToolData.sign | %{ $_.values }) {
+
+        # This is a DLL we sign but do not produce.  Hence it doesn't have a PDB
+        # to be source linked.
```

Comment:
@tmat because we were signing it 😄 

Basically when I converted to SignUtil I started by doing a diagnostic run on our signed builds.  That spit out the list of assemblies which were being signed.  I assumed that was correct and replicated the list in the sign tool data.  

If that's wrong I'm happy to remove.